pkgs/top-level/python-packages.nix

```diff
@@ -2292,6 +2292,9 @@ in {
 
   odfpy = callPackage ../development/python-modules/odfpy { };
 
+  openrazer = callPackage ../development/python-modules/openrazer/pylib.nix { };
+  openrazer-daemon = callPackage ../development/python-modules/openrazer/daemon.nix { };
```

Comment:
Since this is both a python package and an application, you can make an alias `openrazer-daemon = python3.pkgs.openrazer-daemon` in `all-packages.nix` (and then use the alias in the NixOS module).

Reply:
Not sure if I did this correctly.  Please advise if any further changes need to be made regarding this alias.